### PR TITLE
Add PHP 8 support and use github actions for tests

### PR DESCRIPTION
**Pull request by alexander-schranz**
See CI in: https://github.com/alexander-schranz/jackalope-fs/pull/1 -> [checks](https://github.com/alexander-schranz/jackalope-fs/pull/1/checks)

### CODE REVIEW

File: src/Transport/Fs/Filesystem/Storage.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function readBinaryStream($workspace, $path)
         return $this->nodeReader->readBinaryStream($workspace, $path);
     }
 
-    public function readNodeReferrers($workspace, $path, $weak = false, $name)
+    public function readNodeReferrers($workspace, $path, $weak, $name)
```
Comment (alexander-schranz) [inline src/Transport/Fs/Filesystem/Storage.php:76]: Without errors with:

>  Required parameter $name follows optional parameter $weak

File: src/Transport/Fs/Filesystem/Storage/NodeReader.php [hunk 2]
```diff
@@ -138,7 +138,7 @@ public function readBinaryStream($workspace, $path)
         return $res;
     }
 
-    public function readNodeReferrers($workspace, $path, $weak = false, $name)
+    public function readNodeReferrers($workspace, $path, $weak, $name)
```
Comment (alexander-schranz) [inline src/Transport/Fs/Filesystem/Storage/NodeReader.php:141]: Without errors with:

>  Required parameter $name follows optional parameter $weak

**Comment by dantleech** [comment]
:tada:

**Comment by alexander-schranz** [comment]
@dantleech thx for merging. Can we tag this with `0.0.3` this is required by: 

 - https://github.com/phpcr/phpcr-migrations/pull/13

**Comment by dbu** [comment]
i tagged 0.0.3

**Comment by alexander-schranz** [comment]
@dbu Thank you!

**Pull request merged**